src/app/main.py

```diff
@@ -83,6 +84,59 @@ def generate_line_segment(l):
 
     return image.reduceRegions(lines, reducer.setOutputs(band_names), scale)
 
+
+@app.route('/get_image_urls', methods=['GET', 'POST'])
```

Comment:
Should be fine for now, but we need to review naming conventions, to be consistent with raster vs. image vs. map. Dependent projects will have to be updated then.